models/movement.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+// Each Movement needs to have the following features:
+// - A name (required).
+// - A description (no longer than 200 characters).
+// - Starting and ending years (as strings) for the time when the movement
+//    was most prominent.
+// - An 'ancestor', another art movement that this movement descends from.
+const mongoose = require('mongoose');
+
+const movementSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true
+  },
+  description: {
+    type: String,
+  },
+  startYear: {
+    type: String,
+
+  },
+  endYear: {
+    type: String
+  }
+},{
+      timestamps: true,
+      toObject: { virtuals: true },
+      toJSON: { virtuals: true }
+  });
+
+movementSchema.virtual('ancestor').get(function(){
```

Comment:
Not quite. `ancestor` isn't supposed to be a virtual property, it's supposed to be a reference to another Movement! Look back to the notes in `express-api` to see how to set up a relationship in Mongoose.